build/schema.graphql

```diff
@@ -23,3 +25,11 @@ type Account @entity {
     ownedDomains: [Domain!] @derivedFrom(field: "owner")
     createdDomains: [Domain!] @derivedFrom(field: "creator")
 }
+
+type MetaData @entity {
```

Comment:
Metadata is part of the domain, not a separate entity.

The `lockedBy`, `isLocked`, and `metadata` fields should be a part of the domain entity

Reply:
If you look at the subgraph on the graph explorer I tried several different ways  to link the metadata entity into the domain entity last night. It failed every way I tried so I ended up having just merge them all into the domain entity and removed the metadata entity